tsl/src/bgw_policy/policy_utils.h

```diff
@@ -11,5 +11,7 @@
 bool policy_config_check_hypertable_lag_equality(Jsonb *config, const char *json_label,
 												 Oid dim_type, Oid lag_type, Datum lag_datum);
 Datum subtract_integer_from_now(int64 interval, Oid time_dim_type, Oid now_func);
+int64 subtract_integer_from_now_internal(int64 interval, Oid time_dim_type, Oid now_func,
+										 bool *overflow);
 Datum subtract_interval_from_now(Interval *interval, Oid time_dim_type);
```

Comment:
time_utils.c now defines the same function (new changes in master). I will remove subtract_interval_from_now from this file. working on the change.